src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerIEEE14Test.java

```diff
@@ -222,8 +225,8 @@ public void test01GivenCurvesAndEvents() throws IOException {
         List<String> expectedSeriesNames = curveInfosList.stream().map(curveInfos -> curveInfos.getEquipmentId() + "_" + curveInfos.getVariableId()).collect(Collectors.toList());
 
         // get timeseries from mock timeseries db
-        UUID timeSeriesUuid = UUID.fromString(TimeSeriesClientTest.TIME_SERIES_UUID);
-        List<TimeSeries> resultTimeSeries = timeSeriesMockBd.remove(timeSeriesUuid);
+        UUID timeSeriesUuid = TimeSeriesClientTest.TIME_SERIES_UUID;
+        List<TimeSeries> resultTimeSeries = timeSeriesMockBd.get(timeSeriesUuid);
```

Comment:
"timeSeriesMockBd.remove" replaced by "timeSeriesMockBd.get"
is it normal ?

Reply:
It's normal and more logic, this was a mistake...